contrib/tmpfile/podman.conf

```diff
@@ -1,6 +1,6 @@
 # /tmp/podman-run-* directory can contain content for Podman containers that have run
 # for many days. This following line prevents systemd from removing this content.
-x /tmp/podman-run-*
```

Comment:
given podman stores the old tmpdir in the db and will keep using it AFAICT we should keep that old entry as well

Reply:
fixed